libraries/engage/locations/components/FulfillmentSheet/FulfillmentSheet.connector.js

```diff
@@ -19,12 +20,25 @@ function makeMapStateToProps() {
    * @param {Object} props The component props.
    * @returns {Object}
    */
-  return (state, props) => ({
-    locations: getProductLocations(state, props),
-    product: getProduct(state, props),
-    userInput: getUserFormInput(state),
-    fulfillmentPaths: getFulfillmentPaths(state),
-  });
+  return (state, props) => {
+    const {
+      firstName, lastName, mail: email, ...ignoreRest
+    } = getUserData(state);
+
+    return {
+      locations: getProductLocations(state, props),
+      product: getProduct(state, props),
+      userInput: {
```

Comment:
This is a bad way of doing this. This merge should be done inside the `getUserFormInput()` selector, for better performance and memoization. The `mapStateToProps` function shouldn't have side effects.